docs/tutorials/migrate-to-chiselled-rock.rst

```diff
@@ -223,22 +223,22 @@ The output should be similar to:
     2023-04-19 15:56:00.567 :: 2023-04-19 13:56:00.148 Exported to OCI archive 'dotnet-runtime_chiselled_amd64.rock'
 
 At the end of the process, a file named ``dotnet-runtime_chiselled_amd64.rock``
-should be present in the current directory. That's your Chiselled ROCK,
+should be present in the current directory. That's your Chiselled rock,
 as an OCI archive.
 
 
-Test the ROCK
+Test the rock
 -------------
 
-First, import the recently created ROCK into Docker:
+First, import the recently created rock into Docker:
 
 .. literalinclude:: code/migrate-to-chiselled-rock/task.yaml
     :language: bash
     :start-after: [docs:skopeo-copy]
     :end-before: [docs:skopeo-copy-end]
     :dedent: 2
 
-Now inspect the Chiselled .NET Runtime ROCK the same way as it was done for the
+Now inspect the Chiselled .NET Runtime rock the same way as it was done for the
```

Comment:
if you agree with Tiago's note on Chiselled, then this should be changed too.

Reply:
well...I'm not content writer not linguistics expert. I think cases like these are acceptable when you're referring to it as a "thing". In this case, the image is a product.
Anyway, I've applied the suggestion